proseco/guide.py

```diff
@@ -340,6 +349,45 @@ def get_initial_guide_candidates(self):
         return cand_guides
 
 
+
+def check_fid_trap(cand_stars, fids, dither):
+    """
+    Search for guide stars that would cause the fid trap issue and mark as spoilers.
```

Comment:
Add :params: and :returns: to docstring.

Reply:
This is not pretty but done